spotbugs/src/main/java/edu/umd/cs/findbugs/ba/bcp/PatternElement.java

```diff
@@ -211,9 +212,7 @@ protected static BindingSet addOrCheckDefinition(String varName, Variable variab
             bindingSet = new BindingSet(new Binding(varName, variable), bindingSet);
         } else {
             if (!existingVariable.sameAs(variable)) {
-                if (DEBUG) {
-                    System.out.println("\tConflicting variable " + varName + ": " + variable + " != " + existingVariable);
-                }
+                LOG.debug("\tConflicting variable {}: {} != {}", varName, variable, existingVariable);
```

Comment:
not needs tabs?

Reply:
I guess here tab is used to group multiple log entries like `console.group()`?

Reply:
29b7fe000c6e4e6370dce812e60f834379e0502e introduced this tab. I cannot find its intention...